alchemist/alchemist-swingui/src/main/java/it/unibo/alchemist/boundary/gui/isolines/Utilities.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2010-2019, Danilo Pianini and contributors
+ * listed in the main project's alchemist/build.gradle.kts file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.boundary.gui.isolines;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+
+/**
+ * Some utilies for null checking.
+ */
+public final class Utilities {
+
+    private Utilities() {
+    }
+
+    /**
+     * Checks if a variable number of objects are null, in that case an IllegalArgumentException will be thrown.
+     * @throws IllegalArgumentException
+     *              with "null argument" message
+     * @param objects
+     *              the objects to check
+     */
+    public static void requireNonNull(final Object...objects) {
```

Comment:
What an overkill... where is this used?

Reply:
`ConcreteIsolinesFactory` and `ConrecIsolinesFinder` use it. 

When receiving collections or arrays as input, I usually check whether or not they contain null elements. Is this too much?

Reply:
Probably... It may make sense on a "very public" API, where it's likely to see misuse

Reply:
Ok, it's not used anymore